Geometry/CSCGeometryBuilder/src/CSCGeometryParsFromDD.cc

```diff
@@ -159,17 +183,56 @@ bool CSCGeometryParsFromDD::build(const DDCompactView* cview,
            ++it) {
         uparvals.emplace_back(*it);
       }
+
       /** end crap **/
     }
+
     fpar.clear();
-    //    dpar = fv.logicalPart().solid().parameters();
+
     if (fv.logicalPart().solid().shape() == DDSolidShape::ddsubtraction) {
       const DDSubtraction& first = fv.logicalPart().solid();
       const DDSubtraction& second = first.solidA();
       const DDSolid& third = second.solidA();
       dpar = third.parameters();
+      if (std::abs(dpar[0]) < 1.0e-7)
```

Comment:
please, use:
```c++
#include "DataFormats/Math/interface/Rounding.h"
```
and replace lines from #197 to #215 with:
```c++
std::transform(dpar.begin(), dpar.end(), dpar.begin(),
                   [](double i) -> double { return cms_rounding::roundIfNear0(i); });
```

Reply:
Done